aggregator/src/main/java/com/redhat/cloud/notifications/DailyEmailAggregationJob.java

```diff
@@ -34,48 +37,69 @@
     private final EmailAggregationResources emailAggregationResources;
     private final ObjectMapper objectMapper;
 
-    MeterRegistry registry;
-
-    private final AtomicInteger processedGauge;
+    @ConfigProperty(name = "prometheus.pushgateway.url")
+    String prometheusPushGatewayUrl;
 
     @Inject
     @Channel(AGGREGATION_CHANNEL)
     Emitter<String> emitter;
 
-    public DailyEmailAggregationJob(EmailAggregationResources emailAggregationResources, ObjectMapper objectMapper, MeterRegistry registry) {
+    public DailyEmailAggregationJob(EmailAggregationResources emailAggregationResources, ObjectMapper objectMapper) {
         this.emailAggregationResources = emailAggregationResources;
         this.objectMapper = objectMapper;
-        this.registry = registry;
-
-        processedGauge = registry.gauge("daily.email.aggregation.job.processed", new AtomicInteger(0));
     }
 
     @ActivateRequestContext
     public void processDailyEmail() {
-        LocalDateTime now = LocalDateTime.now(UTC);
-        List<AggregationCommand> aggregationCommands = processAggregateEmails(now);
-
-        List<CompletableFuture<Void>> futures = new ArrayList<>();
-        for (AggregationCommand aggregationCommand : aggregationCommands) {
-            try {
-                final String payload = objectMapper.writeValueAsString(aggregationCommand);
-                futures.add(emitter.send(payload).toCompletableFuture());
-            } catch (JsonProcessingException e) {
-                LOG.warn("Could not transform AggregationCommand to JSON object.", e);
+        CollectorRegistry registry = new CollectorRegistry();
+        Gauge duration = Gauge
+                .build()
+                .name("aggregator_job_duration_seconds")
```

Comment:
Metrics names are usually separated with `.`

Reply:
`notifications.aggregator.job.duration.seconds` could help avoiding collisions with another project that would publish metrics starting with `aggregator`. Same idea for other metrics names.